giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py

```diff
@@ -0,0 +1,183 @@
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-03-14 00:00:00',
+                   'weight at time of implant': '28.0 g'},
+     '4222174.1': {'mouse #': '4222174.1', 'species': 'mus musculus', 'ID': 'F4', 'DOB': '2018-10-29 00:00:00',
+                   'genotype': 'Ai94 hemizygous; CaMKII-cre hemizygous; CaMKII-tTA hemizygous', 'sex': 'FEMALE',
+                   'virus injection date': '2019-04-13 00:00:00', 'VIRUS': 'NONE',
+                   'cannula implant date': '2019-04-13 00:00:00', 'weight at time of implant': '23.1 g'},
+     '4222154.1': {'mouse #': '4222154.1', 'species': 'mus musculus', 'ID': 'F5', 'DOB': '2019-01-07 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2019-03-13 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-03-13 00:00:00',
+                   'weight at time of implant': '21.4 g'},
+     '4343702.1': {'mouse #': '4343702.1', 'species': 'mus musculus', 'ID': 'F6', 'DOB': '2019-10-29 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2020-02-20 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2020-02-20 00:00:00',
+                   'weight at time of implant': '19.0 g'},
+     '4222157.4': {'mouse #': '4222157.4', 'species': 'mus musculus', 'ID': 'FD1', 'DOB': '2019-02-08 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'MALE', 'virus injection date': '2019-05-02 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-05-02 00:00:00',
+                   'weight at time of implant': '26.2 g'},
+     '4222169.1': {'mouse #': '4222169.1', 'species': 'mus musculus', 'ID': 'FD2', 'DOB': '2019-03-03 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2019-07-10 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-07-10 00:00:00',
+                   'weight at time of implant': '22.3 g'},
+     '4222169.2': {'mouse #': '4222169.2', 'species': 'mus musculus', 'ID': 'FD3', 'DOB': '2019-03-03 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2019-07-10 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-07-10 00:00:00',
+                   'weight at time of implant': '18.7 g'},
+     '4222169.4': {'mouse #': '4222169.4', 'species': 'mus musculus', 'ID': 'FD4', 'DOB': '2019-03-31 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2019-07-12 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-07-12 00:00:00',
+                   'weight at time of implant': '21.7 g'}}
```

Comment:
this should be included as a csv or excel file

Reply:
i've included a json file instead, makes it easier to look up a subject id specific info as a dicts keys